types/src/on_chain_config/execution_config.rs

```diff
@@ -70,7 +70,11 @@ impl OnChainExecutionConfig {
     /// Features that are ready for deployment can be enabled here.
     pub fn default_for_genesis() -> Self {
         OnChainExecutionConfig::V4(ExecutionConfigV4 {
-            transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
+            transaction_shuffler_type: TransactionShufflerType::Fairness {
+                sender_conflict_window_size: 32,
```

Comment:
While we are at changing the config - can we bump up the default to 256? This will need to bump to maintain the throughput as we increase the concurrency in mainnet after hardware upgrade.